### PR TITLE
[FD-1868] Create the framework

### PR DESCRIPTION
**Pull request by brendagutman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
...
+    # You are ready for installations! 
+    # If you want to deactivate the venv run:
+    deactivate
+    ```
+2. **Install the package** <br>
+    If working on a new feature it is possible to install a package version within
+    the remote or local branch
+      ```
+    # remote
+    pip install git+https://github.com/NIH-NCPI/locutus_utilities.git@{branch_name}
```
Comment (brendagutman) [inline README.md:64]: uninstall reinstall > version

File: src/search_dragon/__init__.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+# Create a logger
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# Create a file handler for logging to a file
+file_handler = logging.FileHandler(LOGS_PATH)
+file_handler.setLevel(logging.INFO)
+file_handler.setFormatter(logging.Formatter(LOGGING_FORMAT))
+
```
Comment (brendagutman) [inline src/search_dragon/__init__.py:18]: remove or look into a little. Bump up from info

File: src/search_dragon/sources/ols_api.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    # If no ontologies - include only the special ontology list
+
+    def format_keyword(self, keywords=None):
+        """
+        Will ensure the keyword is acceptable and apply any fuzzy match
+        q=cancer+brain%20cancer
+        Setting keyword as a list for eventual fuzzy match capability
+        """
+        if keywords is None:
+            keywords = f"Bartonella%20henselae"
```
Comment (brendagutman) [inline src/search_dragon/sources/ols_api.py:45]: Error when no search term

**Review by torstees — APPROVED** [review]
This looks great. Thank you.

**Pull request merged**